src/os/providers/ui.provider.tsx

```diff
@@ -46,7 +46,6 @@ const UIContextProvider = ({
       >
         {antd ? (
           <ConfigProvider
-            prefixCls={appId}
```

Comment:
@thiennv-qng it's not a solution for all apps? what if we need to isolate/custom app themes?

Reply:
@thiennv-qng Let's me cover this bug. It's ok. Please remove the changes in this file.